http.go

```diff
@@ -318,6 +325,19 @@ func NewVideoClient(accountSid string, authToken string, httpClient *http.Client
 	return c
 }
 
+// NewServiceResource returns a new Client to use Service Resource API
+func NewServiceResource(accountSid string, authToken string, httpClient *http.Client) *Client {
+	c := newNewClient(accountSid, authToken, ServiceResourceBaseUrl, httpClient)
+	c.APIVersion = ServiceResourceVersion
+	c.ServiceResourceService = &ServiceResourceService{
+		MessagingService: &MessagingService{c},
+		PhoneNumber:      &PhoneNumberService{c},
```

Comment:
Generally these are plural elsewhere in the API... PhoneNumbers, AlphaSenders, ShortCodes.